framework/source/class/qx/ui/form/RadioButtonGroup.js

```diff
@@ -59,6 +60,7 @@ qx.Class.define("qx.ui.form.RadioButtonGroup",
 
     // attach the listener
     this.__radioGroup.addListener("changeSelection", function(e) {
+      this.fireDataEvent("changeValue", e.getData(), e.getOldData());
       this.fireDataEvent("changeSelection", e.getData(), e.getOldData());
```

Comment:
@mever A fix for this would be to not use an inline function as an event listener, but to change it to a protected override-able member function `_onChangeSelection` .

This way it should be possible to change it to my needs.